email-javamail-composer/build.gradle.kts

```diff
@@ -7,6 +7,8 @@ dependencies {
     api(projects.micronautEmail)
     api(libs.managed.jakarta.mail)
     implementation(mnValidation.micronaut.validation)
+    runtimeOnly(libs.managed.eclipse.angus)
```

Comment:
I think we should move this dependency to `testRuntimeOnly` and tell the user in the documentation they have to add the implementation they choose as a runtimeOnly dependency. 

We could say to you java email you have to add: 

implementation("io.micronaut.email:micronaut-email-javamail")
and: 
runtimeOnly(libs.managed.eclipse.angus)